packages/lib-classifier/src/store/RootStore.js

```diff
@@ -83,12 +83,28 @@ const RootStore = types
       self.subjects.reset()
     }
 
+    function _afterInitialize() {
+      // Because the root store is initialized all the sub-stores are as well
+      // We enable listening between this root store's stores by calling an afterRootInitialize() method on all these sub-stores
+      const allKeys = keys(self);
+      allKeys.forEach(key => {
+        // Filter the current model to only the observable properties of the root store since that's all our models
+        const isFunction = typeof self[key] === 'function';
+        const isVolatile = !isObservableProp(self, key);
+        if (isFunction || isVolatile) return;
+
+        // If the sub-store has the afterRootInitialize() method, run it
+        if (self[key].afterRootInitialize) self[key].afterRootInitialize();
```

Comment:
[minor][nerding] If we wanted, I _think_ we can simplify this like to `self[key].afterRootInitialize?.()`

Reply:
A tip for simplifying the code: This new hook can be avoided entirely by using `autorun` instead of `reaction`.

Reply:
@eatyourgreens - we started with that but because of how the code is executed the `autorun` causes it to try running a tremendous amount of times (essentially, every nextTick is how I think it tries to run again). This is because the parent RootStore isn't reference-able within the stores that are setup within the RootStore.

Reply:
That's weird, because `store.tutorials` reacts to `store.workflows` with `autorun`. Likewise, `store.fieldGuides` reacts to `store.projects` with an `autorun` reaction.

Reply:
For example, this reaction only runs once, if that's helpful.

https://github.com/zooniverse/front-end-monorepo/blob/42e6120a53085352e10bee9b175ecf01a3522c84/packages/lib-classifier/src/store/FieldGuideStore/FieldGuideStore.js#L17-L31